decidim-core/app/jobs/decidim/event_publisher_job.rb

```diff
@@ -32,6 +37,10 @@ def perform(event_name, data)
 
     private
 
+    def event_type
```

Comment:
Is there any event within Decidim that implements the `types` method? How is this supposed to work?

Reply:
Yes, all of them implement them through the concerns `NotificationEvent` and `EmailEvent`. I've made a detailed explanation in the previous comment